.pre-commit-config.yaml

```diff
@@ -43,6 +43,8 @@ repos:
           types-pyYAML,
           types-jsonschema,
           "sentry-kafka-schemas>=1.2.0",
+          "types-protobuf",
+          "sentry-protos>=0.1.74"
```

Comment:
This is FSL, we cannot import it in an Apache 2.0 library

Reply:
Do I understand it right that this is only used for tests? in that case why not write your own protobuf schema?

Reply:
I can do that, I just relied on existing ones out of ease of use